lib/tasks/temporary/env_variables.rake

```diff
@@ -0,0 +1,16 @@
+# Create a .env file from your application.yml file
+task create_dot_env_file: :environment do
```

Comment:
this breaks the second time is run because `config/application.yml` doesn't exist anymore:

```text
Errno::ENOENT: No such file or directory @ rb_sysopen - config/application.yml
/Users/rhymes/Development/forem/lib/tasks/temporary/env_variables.rake:4:in `initialize'
/Users/rhymes/Development/forem/lib/tasks/temporary/env_variables.rake:4:in `open'
```

as it's contained in `bin/setup` which can be called more than once locally (we designed it to be idempotent), we should probably bail from the script if there's no `config/application.html` in the first place

Reply:
Oooooh really good find! Thanks!